### PR TITLE
Voomp submission

### PR DESCRIPTION
**Pull request by SubhamPramanik**
# Application for Registering Team to CODE CAMP 1.0

This pull request template helps you complete an application to the **CODE CAMP 1.0 HACKATHON**. Use the checklist below to verify that you have followed the instructions correctly. 

**(Put x inside [] like [x])**

## Select the type of Pull Request

[] Registration Pull request.

[x] Submission Pull request.

## Checklist ✅  

- [x] I have read the instructions on the README file before submitting my application and gone through [CODE_OF_CONDUCT](). 
- [x] I have Submitted my team's details by making a folder of my team as instructed in [How to send Pull Request page](https://codecamphackathon.netlify.app/makingRegistration/).
- [x] I have used the Markdown file template to add my information for the Hackathon.
- [x] I have given all the necessary details as mentioned in template index.md file and the details are correct and best to my knowledge.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the Details added to the template will be used as a means of communication at the time of result declaration.
- [x] I have added the [event](https://calendar.google.com/calendar/r/eventedit/copy/MHUzM2pvaDQwM3R1cDdxMmdjMmQyYjZpaHAgN3NhMzI0N2FpMzkzczAzbWo0YWMxNWVzdG9AZw) to my calendar.

### CODE REVIEW

File: content/finalist/Voomp/index.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+description: "We at Voomp believe technology enhances every aspect of life and aim make it accessible to everyone."
+Team-Quote: "Driving forward with technology"
+Application_Type: "Mobile Application"
+---
+
+We at Voomp believe technology enhances every aspect of life and aim make it accessible to everyone.
+
+---
+
+# Project's Name
```
Comment (Uyadav207) [inline content/finalist/Voomp/index.md]: mention project's Name is place of project's Name section

File: content/finalist/Voomp/index.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+2. Distribution Service, this will be responsible for serving the content through APIs. Similar to newsapi.org, this will enable 3rd party apps to use our backend for content.
+3. Android application, this is the frontend through which user will be able to consume the content.
+
+---
+
+_**Tech-Stack**_ : Android, Python, Django, PostgreSQL, Docker, AWS
+
+_**GitHub-Link**_ : https://github.com/SubhamPramanik/voomp
+
+_**Hosted-Link**_ : https://drive.google.com/file/d/1lfFNbnTIMi_HxJ0-HyrX7ICiGcC-PRDH/view?usp=sharing and http://ec2-15-207-58-167.ap-south-1.compute.amazonaws.com/api/articles
```
Comment (Uyadav207) [inline content/finalist/Voomp/index.md]: What is this ? Separate the link and also mention which link is for what?

**Review by Uyadav207 — CHANGES_REQUESTED** [review]
Please Do Changes ASAP

**Comment by SubhamPramanik** [comment]
Changed

File: content/finalist/Voomp/index.md (hunk 1 above)
Comment (SubhamPramanik) [reply content/finalist/Voomp/index.md]: Resolved

File: content/finalist/Voomp/index.md (hunk 2 above)
Comment (SubhamPramanik) [reply content/finalist/Voomp/index.md]: Resolved

**Review by Uyadav207 — APPROVED** [review]
Welcome to finals team

**Pull request merged**